bin/main.rb

```diff
@@ -1,58 +1,106 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-puts 'Hello World!'
+class Player
+  attr_reader :name
+  attr_reader :sym
 
-# options for movement
+  def initialize(sym, name)
+    @sym = sym
+    @name = name
+  end
+end
 
-square_spaces = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+class Board
```

Comment:
**[REQUIRED]** 
* All classes are kept in `lib` directory as described in the [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/) [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/lib_dir_exist.png).